templates/docker/conda_install_rapids

```diff
@@ -9,8 +9,8 @@ RUN source activate rapids \
       -c rapidsai-nightly \
       -c nvidia \
       -c ${XGBOOST_CONDA_LABEL} \
-      -c numba \
       -c conda-forge \
+      -c defaults \
       cudatoolkit=${CUDA_MAJORMINOR_VERSION} \
```

Comment:
AFAIK this is redundant and does not change anything as the foundation container has `cudatoolkit` already installed

Doesn't hurt to leave it here, just with `--freeze-installed` it has no impact